hw/ip/keymgr/doc/_index.md

```diff
@@ -178,56 +182,74 @@ An error code register is maintained {{< regref ERR_CODE >}} to check issues tha
-| Reset          | Advance                        | Advance to Initialized state upon completion.                                                                                 |
-| Reset          | Disable / Advance / Generate   | Invalid operation error triggered with no other side effects.                                                                 |
-| Initialized    | Disable / Advance              | Advance to next Disabled state or CreatorRootKey.                                                                             |
-| Initialized    | Generate                       | Invalid operation error triggered with random data, output registers are updated.                                             |
-| CreatorRootKey | Disable / Advance / Generate   | Behave as expected.                                                                                                           |
-| OwnerIntKey    | Disable / Advance / Generate   | Behave as expected.                                                                                                           |
-| OwnerKey       | Disable / Advance / Generate   | Behave as expected.                                                                                                           |
-| Disabled       | Disable / Advance / Generate   | Invalid operation error triggered with random data, both working state and output registers are updated depending on command. |
+| Current State    | Legal Operations               |
+| -------------    | ------------------------------ |
+| Reset            | Advance                        |
+| Initialized      | Disable / Advance              |
+| CreatorRootKey   | Disable / Advance / Generate   |
+| OwnerIntKey      | Disable / Advance / Generate   |
+| OwnerRootKey     | Disable / Advance / Generate   |
+| Invalid/Disabled | None                           |
+
+*  All operations invoked during `Invalid` and `Disabled` states lead to invalid operation error.
+
+### Error Response
+In addition to alerts and interrupts, key manager may also update the working state key and relevant outputs based on current state.
+See the tables below for an enumeration.
+
+| Current State    | Invalid Command | Invalid Output | Invalid Input | Invalid Operation   |
+| -------------    | ----------------| -----------------------------------------------------|
+| Reset            | Not Possible    | Not Possible   | Not possible  | Not updated         |
+| Initialized      | Updated         | Updated        | Not updated   | Not updated         |
+| CreatorRootKey   | Updated         | Updated        | Not updated   | Not updated         |
+| OwnerIntKey      | Updated         | Updated        | Not updated   | Not updated         |
+| OwnerRootKey     | Updated         | Updated        | Not updated   | Not updated         |
```

Comment:
seems no Invalid Operation in these 3 states
```suggestion
| CreatorRootKey   | Updated         | Updated        | Not updated   | Not Possible         |
| OwnerIntKey      | Updated         | Updated        | Not updated   | Not Possible         |
| OwnerRootKey     | Updated         | Updated        | Not updated   | Not Possible         |
```

Reply:
o yeah that's a fair point.